connman/vpn/plugins/wireguard.c

```diff
@@ -226,50 +245,67 @@ static int parse_endpoint(const char *host, const char *port, struct sockaddr_u
 	return 0;
 }
 
-static int parse_address(const char *address, const char *gateway,
-		struct connman_ipaddress **ipaddress)
+static int parse_addresses(const char *address, const char *gateway,
+		GSList **ipaddresses)
 {
+	struct connman_ipaddress *ipaddress = NULL;
 	char buf[INET6_ADDRSTRLEN];
 	unsigned char prefixlen;
+	char **addresses;
 	char **tokens;
 	char *end, *netmask;
 	int err;
+	int i;
 
-	tokens = g_strsplit(address, "/", -1);
-	if (g_strv_length(tokens) != 2) {
-		g_strfreev(tokens);
+	addresses = g_strsplit(address, ", ", -1);
+	if (!g_strv_length(addresses))
 		return -EINVAL;
-	}
 
-	prefixlen = g_ascii_strtoull(tokens[1], &end, 10);
+	for (i = 0; addresses[i]; i++) {
```

Comment:
TODO: limit this to two addresses, IPv4 and IPv6. ConnMan has only one struct for each, so having more than 1 IPv4 and 1 IPv6 is not possible as of yet.